index.html

```diff
@@ -5,32 +5,28 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Odongo full-stack web-developer</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <link rel="stylesheet" href="style.css">
+    <div class="ul-cont">
+        <span class="closeIcon">x</span>
+        <div class="top-nav">
+            <a class="nav-link" href="#Portfolio">Portfolio</a>
```

Comment:
- [ ] please kindly add margin righ to your hamburger icon so it it well positioned and follow the figma design
![Screenshot (681)_LI](https://user-images.githubusercontent.com/65298048/208657370-8777aa83-d7cf-4233-af7f-d2bef30e4536.jpg)